_posts/2019-02-11-lecture-08.md

```diff
@@ -0,0 +1,174 @@
+    
+* $$ P_x(V_i | PA_i) = P(V_i | PA_i) $$ for all $$ V_i \notin X $$ where $$ PA_i $$ represents the parents of $V_i$. That means the conditional distributions of all the nodes not included in the intervention remains the same before and after the intervention. We can see this graphically since while doing intervention, we just delete all the incoming edges into the nodes on which intervention is performed. So the other nodes remain unchanges(in terms of parents)
+
+### Randomized Control Experiments
+We change a controlled variable and observe the change in an outcome variable. All the other variables that influence the outcome variable is either fixed or varied at random. So now any change observed in the outcome variable must have been due to the change in the controlled variable.
+
+Its disadvantage is that its expensive and in many cases impossible to do. The outcome variable can depend on many other variables. So, to find one causal relation, we would need a lot of data and we need to ensure that the other variables are fixed or varied randomly across the experiments which is a very hard task.
+
+### Kidney Stone Experiment
+Lets revisit the kidney stone treatment example. Let R = Recovery, T = Treatment and S = Stone size, then
+
+$$ P(R|T) = \sum_S P(R|S,T)P(S|T) $$
+$$ P(R|do(T)) = \sum_S P(R|S,T)P(S) $$
+
+Note that while intervening, we cut off all incoming edges into the treatment node. So, now S and T are independent
+Figures
+
+### Causal Effect Identifiability
+An aspect of a statistical model is identifiable, if it cannot be changed without there also being some change in the distribution of the observed variables. So if we can express a quantity as a function of the distributions of observed variables, then we can say that quantity is identifiable. Thus the causal effect is identifiable if and only if we can represent $$ P(Y|do(X)) $$ in terms of the distribution of observed variables. i.e. if
+
+$$ P_{M1}(Y|do(X)) = P_{M2}(Y|do(X)) $$ for every pair of models M1 and M2 with the dame graph and the same distibution of observed variables(Unobserved / Latent variable distributions can be changed)
+
+Example - If we have just two random variables X and Y(both observed) where X is the cause of Y and there is no confounder.So, $$ P(Y|do(X)) = P(Y|X) $$ and we know this is identifiable because for changing $$ P(Y|X) $$, we would have to change the distribution of P(X,Y).
+
+But, now lets say we have a confounder C, which is the common cause of both X and Y. And this is not observed. Now, we can change the distribution of C such that P(X,Y) is unchanged but we get different $$ P(Y|do(X)) $$
+$$ P(Y|do(X)) $$ can also be written as $$ P(y | \hat{x}) $$
+#### Back Door Criterion 
+A set of variables Z satisfies the back-door criterion relative to an ordered pair of variables $$ (X_i, X_j) $$ if
```

Comment:
```suggestion
A set of variables Z satisfies the **back-door criterion** relative to an ordered pair of variables $$ (X_i, X_j) $$ if
```
For definitions, please **bold** the thing being defined, here and elsewhere in the file.